.github/workflows/ci.yml

```
@@ -44,6 +44,32 @@ jobs:
       - run: npm ci --legacy-peer-deps
       - run: npm test
 
+  sdk-entrypoint-smoke:
+    name: SDK / ESM & CJS Entry Point Smoke
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: invofi/apps/sdk
+    env:
+      SDK_SMOKE_DIR: ${{ runner.temp }}/invofi-sdk-smoke
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)